src/EVTUI/Program.cs

```diff
@@ -18,6 +19,9 @@ public static AppBuilder BuildAvaloniaApp()
         => AppBuilder.Configure<App>()
             .UsePlatformDetect()
             .WithInterFont()
+#if (OS_WINDOWS)
+            .With(new Win32PlatformOptions { RenderingMode = new Collection<Win32RenderingMode> { Win32RenderingMode.Wgl } })
+#endif

```

Comment:
Just to double-check: you've tested the rendering on both Windows and Linux, and this type of `#if` block allows it to work no matter which you're using? These blocks didn't work in the past, but I assume that's because we weren't defining the relevant variables in the `.csproj` like you've now done.

Reply:
Can confirm that macro works properly for both Windows and Linux.
This line is necessary because by default Avalonia will create an OpenGL ES 3.0 context (on windows ONLY?!?!), which is incompatible with GFD Studio's OpenGL Core 3.3.0 shaders, and I haven't found a documented way of altering this. Asking to switch the backend out to Wgl rather than Angle is all I could find that works, although it's allegedly not recommended.
We might revisit this if we need Angle on Windows and write our own shaders.

Reply:
Why does it work like that 😭 Well... as long as it DOES work, for now!